tests/cases/test_raft.rs

```diff
@@ -209,12 +209,16 @@ impl Interface {
                         is_learner: true,
                         ..Default::default()
                     };
-                    self.mut_prs().insert_learner(*id, progress);
+                    if let Err(e) = self.mut_prs().insert_learner(*id, progress) {
+                        println!("{}", e);
```

Comment:
You can make it `panic` here as well since it is tests.

Reply:
ok